sdk/src/test/java/com/hedera/hashgraph/sdk/AccountAllowanceDeleteTransactionTest.java

```diff
@@ -43,7 +42,7 @@ public static void beforeAll() {
         SnapshotMatcher.start();
     }
 
-    @AfterClass
+    @AfterAll
```

Comment:
This was wrong in many places - missed because JUnit4 was still one the classpath.